src/main.rs

```diff
@@ -72,6 +72,13 @@ struct Cli {
     command: Commands,
 }
 
+#[derive(Parser, Debug)]
+struct S2BasinAndStreamUriArgs {
+    /// Format: s2://{basin}/{stream}
```

Comment:
Can you update the comment to:

```suggestion
    /// S2 URI of the format: s2://{basin}/{stream}
```